### PR TITLE
Text and share changes per request list

### PR DESCRIPTION
**Pull request by ironfroggy**

### CODE REVIEW

File: opendebates/static/less/base.less [hunk 1]
```diff
@@ -113,6 +113,15 @@ body h1 p {
   background-repeat: no-repeat;
 }
 
+// Temporary
```
Comment (dpoirier) [inline opendebates/static/less/base.less]: Should we make these changes just for florida?

**Comment by vkurup** [comment]
Looks good to me. Note Dan's question, which I'd consider a nice-to-have-if-its-not-too-much-work. Or at minimum a loud FIXME comment so we know which things to change if we deploy a presidential site.

**Comment by ironfroggy** [comment]
Moved to the avoid the FIXME and also added the email text.

**Comment by vkurup** [comment]
I'm not seeing the CSS overrides at all in the PR anymore

**Comment by vkurup** [comment]
I'm not seeing the florida changes, even after adding the less file. I think `SITE_THEME` needs to be `SITE_THEME_NAME` on line 26 of base.html

**Comment by vkurup** [comment]
:shipit:

**Pull request merged**